packages/core/src/components/Sequence/Sequence.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { Fragment, FC, useState } from 'react';
+import {
+  Stepper,
+  Step,
+  StepContent,
+  StepLabel,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import SequenceFooter from './SequenceFooter';
+
+const useStyles = makeStyles((theme: any) => ({
+  content: {
+    padding: theme.spacing(0, 4, 4, 4),
+    backgroundColor: 'white',
```

Comment:
Think this should be `theme.palette.background.default`? Or maybe not set at all